<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>druzynyForm</class>
 <widget class="QWidget" name="druzynyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drużyny</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_usun">
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>141</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nazwa drużyny:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_dodaj">
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_losuj_drzewko">
     <property name="text">
      <string>Losuj drzewko</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ilość zawodników:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
